### PR TITLE
[INT-173] Add Resume entity.

### PR DESCRIPTION
**Pull request by polina-koval**
https://huntflow.atlassian.net/browse/INT-173

### CODE REVIEW

**Review by KlochkovHF — APPROVED** [review]

File: huntflow_api_client/entities/resume.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        applicant_id: int,
+        external_id: int,
+    ) -> ApplicantResumeResponse:
+        """
+        API method reference
+            https://api.huntflow.ai/v2/docs#get-/accounts/-account_id-/applicants/-applicant_id-/externals/-external_id-
+
+        :param account_id: Organization ID
+        :param applicant_id: Applicant ID
+        :param external_id: External ID
```
Comment (alexreznikoff) [inline huntflow_api_client/entities/resume.py:25]: :param external_id: Resume ID

File: huntflow_api_client/entities/resume.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        :return: An applicant resume
+        """
+        response = await self._api.request(
+            "GET",
+            f"/accounts/{account_id}/applicants/{applicant_id}/externals/{external_id}",
+        )
+        return ApplicantResumeResponse.parse_obj(response.json())
+
+    async def get_resume_sources(self, account_id: int) -> ApplicantSourcesResponse:
```
Comment (alexreznikoff) [inline huntflow_api_client/entities/resume.py:34]: get_sources

File: huntflow_api_client/entities/resume.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
+from huntflow_api_client.entities.base import (
+    BaseEntity,
+    DeleteEntityMixin,
+    GetEntityMixin,
+    UpdateEntityMixin,
+)
+from huntflow_api_client.models.request.applicants import ApplicantResumeUpdateRequest
```
Comment (alexreznikoff) [inline huntflow_api_client/entities/resume.py:7]: move it to `huntflow_api_client.models.request.resume`

File: huntflow_api_client/entities/resume.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+from huntflow_api_client.models.response.applicants import ApplicantSourcesResponse
```
Comment (alexreznikoff) [inline huntflow_api_client/entities/resume.py:8]: move it to `huntflow_api_client.models.response.resume`

File: huntflow_api_client/entities/resume.py [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        applicant_id: int,
+        external_id: int,
+    ) -> None:
+        """
+        API method reference
+            https://api.huntflow.ai/v2/docs#delete-/accounts/-account_id-/applicants/-applicant_id-/externals/-external_id-
+
+        :param account_id: Organization ID
+        :param applicant_id: Applicant ID
+        :param external_id: External ID
```
Comment (alexreznikoff) [inline huntflow_api_client/entities/resume.py:57]: :param external_id: Resume ID

File: huntflow_api_client/entities/resume.py [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        external_id: int,
+        data: ApplicantResumeUpdateRequest,
+    ) -> ApplicantResumeResponse:
+        """
+        API method reference
+            https://api.huntflow.ai/v2/docs#put-/accounts/-account_id-/applicants/-applicant_id-/externals/-external_id-
+
+        :param account_id: Organization ID
+        :param applicant_id: Applicant ID
+        :param external_id: External ID
```
Comment (alexreznikoff) [inline huntflow_api_client/entities/resume.py:77]: :param external_id: Resume ID

**Review by alexreznikoff — APPROVED** [review]

**Pull request merged**